src/games/strategy/triplea/ui/ExtendedStats.java

```diff
@@ -68,11 +68,9 @@ private void fillExtendedStats(final GameData data) {
         m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
       }
       // add number of techs
-      if (true) {
-        final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
-        statsExtended.add(new TechCountStat());
-        m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
-      }
+      final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
```

Comment:
@DanVanAtta Ah I know now why this is... Due to the move of this variable one block up in terms of a hierarchy, some lines below, a second variable with the exact same name is declared... Just rename one of those 2 variables